### PR TITLE
Fixes ALL problems (visible through vsc)

### PR DESCRIPTION
**Pull request by SandPoot**
## About The Pull Request
Title.

## Why It's Good For The Game
This is probably not and very bad, test it and make sure it absolutely works before merging.

## Changelog
:cl:
code: Fixed all visible problems in visual studio code, should have no in-game effect.
/:cl:

### CODE REVIEW

File: code/modules/admin/ToRban.dm [hunk 1]
```diff
@@ -38,9 +38,9 @@
 			F["last_update"] << world.realtime
 			log_misc("ToR data updated!")
 			if(usr)	usr << "ToRban updated."
-			return 1
+			return
```
Comment (valzargaming) [inline code/modules/admin/ToRban.dm:41]: But why though? If I ran this proc manually I'd want to know if it succeeded or not...

**Review by valzargaming — COMMENTED** [review]
What exactly was changed here other than some bit flags being fixed to actual operators / vice versa, and return values being altered for no apparent reason?

File: code/modules/admin/ToRban.dm (hunk 1 above)
Comment (SandPoot) [reply code/modules/admin/ToRban.dm:41]: spacemandmm says that using a spawn messes up the result either way, and in any case, there's a `usr << "ToRban updated."` there

**Comment by SandPoot** [comment]
> What exactly was changed here other than some bit flags being fixed to actual operators / vice versa, and return values being altered for no apparent reason?

Like the pr says, it tries to resolve the problems that were appearing in vsc, allowing you to focus on actual problems that you might have caused with your work.

**Comment by valzargaming** [comment]
I'm closing this because I'm seeing a lot of changes in here that have no justification other than "I just did what visual studio code told me to." Seemingly random return values were removed and bitwise functions were changed to logical functions, which would either change the semantics of the code or at the very least make debugging harder because expected return values would not exist.

**Pull request closed without merging**